src/docs/patterns/lists.mdx

```diff
@@ -60,7 +197,142 @@ Occasionally, you may want users to be able to re-order the items in a list manu
 
-Tables are similar to lists but are slightly more structured by ordering data for a list item into columns. These work well on larger screens, but at a mobile size, they should collapse into a list for better formatting.
+Lists emphasize individual items, while tables highlight a comparison between each data point. However, tables generally perform poorly on a mobile device. Do not structure the list as a table if you don't expect your users examining the data "vertically".
 
-<MaterialDesignDescription title={'Data Tables'} url={'https://material.io/components/data-tables/'} />
+<React.Fragment>
+    <Divider />
+</React.Fragment>
+
+# Behaviors
+
+Lists can be interactive. In general, there are two types of interactions: global list actions and local list actions.
+
+<DemoCard
+    repository={'action-list'}
+    angular={
+        'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Faction-list%2Faction-list.component.ts'
+    }
+    react={
+        'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/action-list/index.tsx&initialpath=action-list'
+    }
+    ionic
+    reactNative
+/>
+
+## Global List Actions
+
+Global list actions are the actions that manipulate the entire list, rather than treating each list item individually. The buttons typically sit in the [Appbar](/patterns/appbar), the panel header, or a button row (desktop-only).
+
+<ImageGrid
+    images={[GlobalActionAppBarImage, GlobalActionPanelHeaderImage, GlobalActionAsButtonImage]}
+    caption={[
+        `1: Global actions in the Appbar.`,
+        '2: Global actions in the panel header.',
+        '3: Global actions in its own button row.',
+    ]}
+/>
+
+Common global list actions include:
+
+-   Adding a new list item
+-   Sorting and filtering the list
+-   Toggling a different view, such as a grid view
+-   Exporting or downloading the list data
+
+### Drag and Drop
+
+<ImageGrid images={[SortableListImage]} regularWidth />
+<DemoCard repository={'sortable-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fsortable-list%2Fsortable-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/sortable-list/index.tsx&initialpath=sortable-list'} ionic reactNative />
+
+Drag and drop is a special case for list sorting. It re-orders the items in a list manually. This should be accomplished by placing a drag handle as the leftmost element in the row. The ability to re-order the list should be locked behind an Edit Mode toggle mechanism.
+
+## Local List Actions
+
+<ImageGrid
+    images={[LocalListActionsImage]}
+    caption={[
+        '1: A more_vert icon, used to open a menu for more list actions.',
+        '2: A chevron, usually opens up a new page with list item details.',
```

Comment:
I think normally the chevron is just an icon and the entire row is clickable to navigate to the new page. I don't think we would typically see a list-item that has actions and a chevron (like number 4).

We may want to flesh this out a bit.

Reply:
good point. I will take out #4 and treat chevron a "special icon" that indicates the navigation you mentioned above.

Reply:
Changed the caption to 

"2: A chevron, usually indicates the entire list item is clickable and opens up a new page with list item details."